sagemaker-debugger/pytorch_profiling/pt-resnet-profiling-multi-gpu-multi-node.ipynb

```diff
@@ -9,14 +9,37 @@
     "This notebook will walk you through creating a PyTorch training job with the SageMaker Debugger profiling feature enabled. It will create a multi GPU multi node training.\n"
```

Comment:
<p>To enable profiling, you need to pass the Debugger profiling configuration (<code>profiler_config</code>), a list of Debugger rules (<code>rules</code>), and the image URI (<code>image_uri</code>) to the estimator. Debugger enables monitoring and profiling while the SageMaker estimator requests a training job.</p><p> </p><p> </p><p>^ please apply this throughout the other notebooks too.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done.